rustc_codegen_spirv/src/link.rs

```diff
@@ -1,27 +1,31 @@
 use crate::{linker, SpirvCodegenBackend, SpirvModuleBuffer, SpirvThinBuffer};
-use rustc_codegen_ssa::back::lto::{LtoModuleCodegen, SerializedModule, ThinModule, ThinShared};
-use rustc_codegen_ssa::back::write::CodegenContext;
-use rustc_codegen_ssa::CodegenResults;
-use rustc_data_structures::owning_ref::OwningRef;
-use rustc_data_structures::rustc_erase_owner;
-use rustc_data_structures::sync::MetadataRef;
-use rustc_errors::{DiagnosticBuilder, FatalError};
-use rustc_middle::bug;
-use rustc_middle::dep_graph::WorkProduct;
-use rustc_middle::middle::cstore::NativeLib;
-use rustc_middle::middle::dependency_format::Linkage;
-use rustc_session::config::{CrateType, Lto, OutputFilenames, OutputType};
-use rustc_session::output::{check_file_is_writeable, invalid_output_for_target, out_filename};
-use rustc_session::utils::NativeLibKind;
-use rustc_session::Session;
-use std::collections::HashSet;
-use std::env;
-use std::ffi::{CString, OsStr};
-use std::fs::File;
-use std::io::{Read, Write};
-use std::path::{Path, PathBuf};
-use std::process::Command;
-use std::sync::Arc;
+use rustc_codegen_ssa::{
+    back::{
+        lto::{LtoModuleCodegen, SerializedModule, ThinModule, ThinShared},
+        write::CodegenContext,
+    },
+    CodegenResults,
+};
+use rustc_data_structures::{owning_ref::OwningRef, rustc_erase_owner, sync::MetadataRef};
+use rustc_errors::FatalError;
+use rustc_middle::{
+    bug, dep_graph::WorkProduct, middle::cstore::NativeLib, middle::dependency_format::Linkage,
+};
+use rustc_session::{
+    config::{CrateType, Lto, OutputFilenames, OutputType},
+    output::{check_file_is_writeable, invalid_output_for_target, out_filename},
+    utils::NativeLibKind,
+    Session,
+};
+use std::{
+    collections::HashSet,
+    env,
+    ffi::{CString, OsStr},
+    fs::File,
+    io::{Read, Write},
+    path::{Path, PathBuf},
+    sync::Arc,
+};
```

Comment:
wow, what a rustfmt `merge_imports = true`! is this standard across embark repos, or was this accidental? (not sure if it's anywhere else in this diff, being consistent might be good)

Reply:
Hah no, this was just a manual drive by change when I was changing this file, I can revert if you feel like it's noise.

Reply:
I thiiink I'd prefer it reverted, if it's not too much work? Like, stylistically, I don't mind either way, but what I do mind is consistency, and everywhere else has it separated. (I realllly want the [`module`](https://github.com/rust-lang/rustfmt/issues/3362#issuecomment-642564290) rustfmt merge_imports option to stabilize, then we could set it in rustfmt.toml...)

Reply:
No worries, easy to revert, I just get sad when I see duplicates in module imports. :wink: